src/pip/_internal/req/__init__.py

```diff
@@ -46,7 +46,6 @@ def install_given_reqs(
 
     (to be called after having downloaded and unpacked the packages)
     """
-
```

Comment:
Seems like that a newline was removed from this file by accident. You can re-add it so that this change doesn't show up as part of the PR.

Reply:
Eh! yeah, I'll update it in subsequent commits.